waterlog/src/Stylesheets/_segmentLeak.scss

```diff
@@ -79,6 +85,14 @@
     height: 400px;
     margin: 25px 0;
 }
+.segment-map2 {
+    .segment-map {
+        height: 200px;
```

Comment:
line 85 is 400 and this is 200? why such large px? on what was this tested?

Reply:
i remove the second one  with 200 its  was for testing purposes ,its not used